pkg/controller/component/reclaimer/log.go

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2021 Vesoft Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package reclaimer
+
+import (
+	"github.com/go-logr/logr"
+
+	"github.com/vesoft-inc/nebula-operator/pkg/logging"
+)
+
+var _log = logging.Log.WithName("controller")
```

Comment:
reclaimer module append name "reclaimer" after "controller"?

Reply:
Which packages should be added to `WithName`?

1. Only top package
2. Only bottom package
3. Every package
4. Top and bottom packages

And I think 4 is better, it's be neither too many levels of packages nor too few.